pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -505,6 +512,116 @@ func (r *ReconcileCnsRegisterVolume) Reconcile(ctx context.Context,
 					storageClassName, vc.Config.Host, err)
 				return reconcile.Result{RequeueAfter: timeout}, nil
 			}
+
+			cnsVolumeInfoCRList := &cnsvolumeinfov1alpha1.CNSVolumeInfoList{}
+			restConfig, err := clientConfig.GetConfig()
+			if err != nil {
+				log.Errorf("failed to get Kubernetes config. Err: %+v", err)
+				return reconcile.Result{RequeueAfter: timeout}, nil
+			}
+			cnsOperatorClient, err := k8s.NewClientForGroup(ctx,
+				restConfig, apis.GroupName)
+			if err != nil {
+				log.Errorf("failed to create cns operator client. Err: %v", err)
+				return reconcile.Result{RequeueAfter: timeout}, nil
+			}
+
+			err = cnsOperatorClient.List(ctx, cnsVolumeInfoCRList, client.InNamespace(common.GetCSINamespace()))
+			if err != nil {
+				log.Errorf("failed to list %q CR. Error: %+v", cnsvolumeinfo.CRDSingular, err)
+				return reconcile.Result{RequeueAfter: timeout}, nil
+			}
+			namespace := instance.Namespace
+			storagePolicyUsageList := &storagepolicyusagev1alpha1.StoragePolicyUsageList{}
+			err = cnsOperatorClient.List(ctx, storagePolicyUsageList, client.InNamespace(namespace))
+			if err != nil {
+				log.Errorf("failed to list %s CR from supervisor namespace %q. Error: %+v",
+					storagepolicyusagev1alpha1.CRDSingular, namespace, err)
+				return reconcile.Result{RequeueAfter: timeout}, nil
+			}
+			foundStoragePolicyUsageCR := false
+			storagePolicyUsageCR := &storagepolicyusagev1alpha1.StoragePolicyUsage{}
+			patchedStoragePolicyUsageCR := &storagepolicyusagev1alpha1.StoragePolicyUsage{}
+			if len(storagePolicyUsageList.Items) > 0 {
+				for _, item := range storagePolicyUsageList.Items {
```

Comment:
Why not use Get with name of StoragePolicyUsage instance?

Reply:
Done